apps/ledger-live-mobile/src/components/RequiresBLE/BluetoothDisabled.tsx

```diff
@@ -24,13 +26,13 @@ function BluetoothDisabled() {
   return (
     <SafeAreaContainer>
       <IconBox Icon={BluetoothMedium} iconSize={24} boxSize={64} />
-      <Text variant={"h2"} mb={5} mt={7}>
+      <Text variant={"h2"} mb={5} mt={7} textAlign="center">
```

Comment:
If you want to improve this, you have a component from the ui drawer that have the exact same layout (icon box, title, description)
It's `ModalHeader`